packages/@ember/-internals/metal/lib/cached.ts

```diff
@@ -111,6 +113,8 @@ export const cached: MethodDecorator = (...args: any[]) => {
     throwCachedGetterOnlyError(key);
   }
 
+  CacheMap.set(target, [...(CacheMap.get(target) || []), key]);
```

Comment:
should this be a WeakMap of WeakSets? rather than a WeakMap of Arrays?

Reply:
changed to set instead of array. weakset does not work, since we are saving pure values (string)